mifospay/src/main/res/layout/fragment_history.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.design.widget.CoordinatorLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this change really needed?

Reply:
It is not a change I made manually. Its GitHub glitch

Reply:
> It is not a change I made manually. Its GitHub glitch

Actually these are because of Android Studio's auto linting. You can fix this by editing this file in other text editor or manually changing your Android studio specifications

Reply:
@Bhavnaharitsa Ma'am, github isn't glitchy, as @devansh-299 pointed out, you can change this in editor preferences and by setting codestyle to the `Project` Style indentations

Reply:
Okay, shall keep a note of it